runall.sh

```diff
@@ -0,0 +1,4 @@
+for benchmark in 2DConvolution 2mm 3mm atax bicg blocked_transform covariance gemm gesummv gramschmidt matmulchain mvt syr2k syrk; do
```

Comment:
is this required ?

Reply:
No, I did not touch this file, maybe this PR needs rebasing.

Reply:
right, can you please rebase? I removed this file in sycl-mlir, so all changes are through PRs.

Reply:
Rebased now.